apprise/utils.py

```diff
@@ -102,7 +102,7 @@
 NOTIFY_CUSTOM_COLON_TOKENS = re.compile(r'^:(?P<key>.*)\s*')
 
 # Used for attempting to acquire the schema if the URL can't be parsed.
-GET_SCHEMA_RE = re.compile(r'\s*(?P<schema>[a-z0-9]{2,9})://.*$', re.I)
+GET_SCHEMA_RE = re.compile(r'\s*(?P<schema>[a-z0-9]{2,10})://.*$', re.I)
```

Comment:
is it maybe easier to just call this addition `schan` instead of `serverchan`.  Then this change isn't nessisary? Or do you prefer having the full `serverchan://` url for consistency?  Thoughts?

Reply:
You're right. The full name has appeared in the introduction. It's more convenient to simplify the protocol name